std/random.d

```diff
@@ -2397,40 +2584,7 @@ Returns:
     Range whose elements consist of the elements of $(D r),
     in random order.  Will be a forward range if both $(D r) and
     $(D rng) are forward ranges, an input range otherwise.
-
-Example:
-----
-int[] a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8 ];
-foreach (e; randomCover(a))
-{
-    writeln(e);
-}
-----
-
-$(B WARNING:) If an alternative RNG is desired, it is essential for this
-to be a $(I new) RNG seeded in an unpredictable manner. Passing it a RNG
-used elsewhere in the program will result in unintended correlations,
-due to the current implementation of RNGs as value types.
-
-Example:
-----
-int[] a = [ 0, 1, 2, 3, 4, 5, 6, 7, 8 ];
-foreach (e; randomCover(a, Random(unpredictableSeed)))  // correct!
-{
-    writeln(e);
-}
-
-foreach (e; randomCover(a, rndGen))  // DANGEROUS!! rndGen gets copied by value
-{
-    writeln(e);
-}
-
-foreach (e; randomCover(a, rndGen))  // ... so this second random cover
-{                                    // will output the same sequence as
-    writeln(e);                      // the previous one.
-}
```

Comment:
This is probably the only "controversial" bit. While std.random is far from perfect, this particular nastiness isn't there anymore as now the overloads are `auto ref`:

https://run.dlang.io/is/6N3Nb9